app/healthcheck.go

```diff
@@ -59,5 +59,8 @@ func (app *App) engineCycle(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	io.WriteString(w, app.Engine.NextCycleID().String())
+	if _, err := io.WriteString(w, app.Engine.NextCycleID().String()); err != nil {
+		http.Error(w, "engine unable to move to next cycle", http.StatusBadRequest)
+		return
+	}
```

Comment:
This is incorrect; It's also not a bad request, and the error comes from the `Write` method being unable to write to the `ResponseWriter` (presumably because the other end canceled the request) and not from the engine.

An error in writing a response to a request that's already been canceled should be ignored. As noted in a previous PR.

Reply:
sure thing.  I'll set it to ignore the error if you're ok with it (I decided to err on the side of caution).